codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/presignable/PresignableUrlIntegration.kt

```diff
@@ -144,6 +150,27 @@ class PresignableUrlIntegration(private val presignedOperations: Map<String, Set
         }
     }
 
+    private fun renderPresignURLAPIInServiceClient(
+        writer: SwiftWriter,
+        clientName: String,
+        op: OperationShape,
+        inputType: String
+    ) {
+        writer.apply {
+            openBlock("extension $clientName {", "}") {
+                val params = listOf("input: $inputType", "expiration: Foundation.TimeInterval")
+                val returnType = "ClientRuntime.URL"
```

Comment:
What is `ClientRuntime.URL`?  Why can't this just be Foundation's URL type?

Reply:
It's a primitive type extension of `Foundation.URL` as seen here: [link](https://github.com/smithy-lang/smithy-swift/blob/53fbde27f536f4287edfd4ff08bb5ebe8dc75169/Sources/ClientRuntime/PrimitiveTypeExtensions/URL%2BExtension.swift#L8), and this is the type returned by the wrapped method in `Models.swift`, as seen here: [link](https://github.com/awslabs/aws-sdk-swift/blob/8d8bacaf1cd81e9d7534a1ea28fd525bf6e179c3/Sources/Services/AWSS3/models/Models.swift#L10664)

Reply:
Given that the extension should be applied anyways, will change the return type of wrapper method to `Foundation.URL`.